validator_client/src/attestation_service.rs

```diff
@@ -431,7 +431,7 @@ impl<T: SlotClock + 'static, E: EthSpec> AttestationService<T, E> {
             .unzip();
 
         if attestations.len() == 0 {
-            warn!("WARN: No attestations were published");
+            warn!(log, "WARN: No attestations were published");
```

Comment:
The `warn!` macro already adds the `WARN` part of the log, so we don't need to add it explicitly

```suggestion
            warn!(log, "No attestations were published");
```

Reply:
> The `warn!` macro already adds the `WARN` part of the log, so we don't need to add it explicitly

Removed the `WARN` and used `is_empty()` for improved readability.